Tests/test_file_ico.py

```diff
@@ -229,3 +229,8 @@ def test_draw_reloaded(tmp_path):
 
     with Image.open(outfile) as im:
         assert_image_equal_tofile(im, "Tests/images/hopper_draw.ico")
+
+
+def test_empty():
+    with pytest.raises(SyntaxError):
+        IcoImagePlugin.IcoImageFile("Tests/images/no_icons.ico")
```

Comment:
```suggestion
    with pytest.raises(SyntaxError) as e:
        IcoImagePlugin.IcoImageFile("Tests/images/no_icons.ico")
    assert str(e.value) == "No images were found"
```
There was already a SyntaxError raised before your code change, so this alters the test to check the message.

Reply:
Good point – changed to `with pytest.raises(SyntaxError, match="No images were found"):` :)